Deferred/DispatchBlockFlags.swift

```diff
@@ -10,24 +10,29 @@ import Dispatch
 
 extension dispatch_block_flags_t: RawOptionSetType {
 
+    /// The empty options set.
     public static var allZeros: dispatch_block_flags_t {
         return dispatch_block_flags_t(0)
     }
 
+    /// Convert from a bitmask of `UInt`.
     public init(rawValue: UInt) {
         self.init(rawValue)
     }
 
+    /// Create an instance initialized with `nil`.
```

Comment:
The docs on this differ in style from the docs in the rest of Deferred. These use an imperative ("Convert!", "Create!"), the others use a third-person indicative ("Converts", "Creates").